x/feeburner/client/cli/tx.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/neutron-org/neutron/x/feeburner/types"
+)
+
+var DefaultRelativePacketTimeoutTimestamp = uint64((time.Duration(10) * time.Minute).Nanoseconds())
```

Comment:
Please remove this file, we do not have txs for this module so it maybe confusing to have it in the tx section of the cli.